jobserver/views/job_requests.py

```diff
@@ -215,9 +221,24 @@ def get(self, request, *args, **kwargs):
             )
         )
 
+        # build up is_missing_updates to define if we've not seen the backend
+        # running this JobRequest for a while.
+        # it's completed, we don't expect updates now
+        incomplete = not job_request.is_completed
+
+        # was the last update more than our threshold ago?  The last_updated_at
+        # annotation uses a default of `now` so we don't have to deal with None
+        # here.
+        delta = timezone.now() - job_request.last_updated_at
+        threshold = timedelta(minutes=30)
```

Comment:
We've now used "30" in both Job Requests and Jobs. Might be worth extracting into a constant in core.py as `LAST_UPDATE_THRESHOLD` is similar?